src/content/api/loaders.md

```diff
@@ -471,3 +509,96 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+There are some ways to throw out errors from inside a loader:
+
+### Using `this.emitError`
+
+It just report errors but not interrupt module compiling.
+
+see [this.emitError](/api/loaders/#this-emiterror).
+
+### Using `throw` (or other uncaught exception)
+
+Throw out an uncaught error while loader are running will cause current module compiling failed.
+
+For example:
+
+__./src/index.js__
+
+```js
+
+require('./loader!./lib');
+
+```
+
+__./src/loader.js__
+
+```js
+
+module.exports = function(source){
+  throw new Error('Here is an fatal Error!');
+};
+
+```
+
+the module that ouccur this error will be packed into bundle (as an error module) like the following:
```

Comment:
Not sure what you mean on this line

Reply:
The module on which loader had thrown this error will get bundled like this: